privacyidea/lib/utils/__init__.py

```diff
@@ -1147,52 +1196,29 @@ def check_pin_policy(pin, policy):
     :param policy: The policy that describes the allowed contents of the PIN.
     :return: Tuple of True or False and a description
     """
-    chars = {"c": r"[a-zA-Z]",
-             "n": r"[0-9]",
-             "s": r"[\[\].:,;_<>+*!/()=?$§%&#~^-]"}
+
     ret = True
     comment = []
 
     if not policy:
         return False, "No policy given."
+    if not pin:
```

Comment:
Please remove this again, since it changes the logic.

Reply:
done